ui.apps/src/main/content/jcr_root/apps/acs-commons/renderconditons/iscurrentusermemberof/iscurrentusermemberof.jsp

```diff
@@ -120,7 +120,7 @@
                 }
             }
         } catch (Exception ignored) {
-            log.debug("Ignored exception occurred while checking iscurrentusermemberof.jsp isUserMemberOf(authorizable, groups)", ignored.getMessage());
+            // log.debug("Ignored exception occurred while checking iscurrentusermemberof.jsp isUserMemberOf(authorizable, groups)", ignored.getMessage());
```

Comment:
I think passing the log as argument is preferable over just removing it altogether.